cluster/saltbase/salt/kube-addons/kube-addon-update.sh

```diff
@@ -481,6 +481,7 @@ function update-addons() {
     # That's why we pass an empty string as the version separator.
     # If the description differs on disk, the object should be recreated.
     # This is not implemented in this version.
+    reconcile-objects ${addon_path} Deployment "" &
```

Comment:
How is this going to update Deployments? AFAIU name for an object will not change, so how addon-updater will realise it has to do an update? Has this been tested?